app/src/main/java/com/arygm/quickfix/model/account/AccountRepositoryFirestore.kt

```diff
@@ -83,23 +83,6 @@ class AccountRepositoryFirestore(private val db: FirebaseFirestore) : AccountRep
         }
   }
 
-  private fun performFirestoreOperation(
-      task: Task<Void>,
-      onSuccess: () -> Unit,
-      onFailure: (Exception) -> Unit
-  ) {
-    task.addOnCompleteListener { result ->
-      if (result.isSuccessful) {
-        onSuccess()
-      } else {
-        result.exception?.let { e ->
-          Log.e("AccountRepositoryFirestore", "Error performing Firestore operation", e)
-          onFailure(e)
-        }
-      }
-    }
-  }
-
```

Comment:
Nice job in doing so because we use this specifc function in every repository class.